WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -55,6 +56,7 @@ + (NSArray *)routes
 #endif
     [[FBRoute POST:@"/wda/pressButton"] respondWithTarget:self action:@selector(handlePressButtonCommand:)],
     [[FBRoute POST:@"/wda/siri/activate"] respondWithTarget:self action:@selector(handleActivateSiri:)],
+    [[FBRoute POST:@"/wda/launchUnattachedApp"].withoutSession respondWithTarget:self action:@selector(handleLaunchUnattachedApp:)],
```

Comment:
there is already an endpoint to launch apps. Can we just add another option to it?

Reply:
A duplicate endpoint with `withoutSession` is possible, too, as in [lock](https://github.com/appium/WebDriverAgent/blob/master/WebDriverAgentLib/Commands/FBCustomCommands.m#L42-L43)

Reply:
I think @imurchie makes a good point: this functionality belongs in a `withoutSession` endpoint.      However, don't you think this is not explicit enough? How would a user guess that passing the session argument into `wda/apps/launch` will result the app dying when WDA dies, but that not passing the `session` will result in the app surviving the death of WDA? On the other hand, if we take another argument into `wda/apps/launch`, then won't we have a kind of duplication? In order to launch an unattached app the user will have to both pass `unattached = true` and not pass `session`. I'm happy to do what you think is best.

Reply:
I got you point. What if we rename the endpoint to `wda/apps/launchUnattached` to just follow the common approach?

Reply:
Done